staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go

```diff
@@ -32,6 +33,34 @@ import (
 	"k8s.io/client-go/dynamic"
 )
 
+var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
+
+func randSeq(n int) string {
```

Comment:
Try using `k8s.io/apiserver/pkg/storage/names.SimpleNameGenerator.GenerateName` instead.  That way you can be sure to start with a letter which is a validation requirement.